README.rst

```diff
@@ -1,4 +1,4 @@
-|mailchimp3 v3.0.12 on PyPi| |MIT license| |Stable|
+|mailchimp3 v3.0.13 on PyPi| |MIT license| |Stable|
```

Comment:
The link here is defined at the bottom and needs to be updated down there as well

Reply:
Yikes, sorry about that. This has been updated. Thanks again!